doc/en/api/1.0.0/styles.yaml

```diff
@@ -574,30 +605,35 @@ paths:
     post:
       operationId: postLayerStyles
       summary: Add a new style
-      description: Adds a new style entry to the layer. Note that using the XML content only adds the entry to the catalog and does not upload style content. Use POST with a ZIP file to create and upload at once, or POST the XML content and then separately PUT the style info at /layers/{layer}/styles/{style}.
+      description: Adds a new style entry to the layer. The style named in styleBody must alread exist, and will not be altered by this request. 
       parameters:
         - name: layer
           in: path
           required: true
           description: Name of the layer to manage styles for
           type: string
         - name: styleBody
-          description: Style body information to upload.
+          description: Style body information naming an existing style to add to the layer
           in: body
           required: true
           schema:
             $ref: "#/definitions/StyleInfoPost"
+        - name: default
+          type: boolean
+          in: query
+          description: Whether to make this the default style for the layer.
+          default: false
+          required: false
       consumes:
         - application/xml
         - application/json
-        - application/zip
```

Comment:
was was zip formally supported and then no longer works? or was the functionality never there ...

Reply:
Never there